ebucore/amagi_ebucoreExtension.ttl

```diff
@@ -375,6 +375,41 @@
     rdfs:label    "releaseCountry"@en ;
     skos:definition "This property denotes the country where the asset is released."@en .
 
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint>
+    rdf:type		rdfs:Class ;
+    rdf:label		"Cuepoint"@en ;
+    rdfs:comment 	"To provide cuepoint metadata for an asset."@en ;
+    rdfs:subClassOf	owl:Thing .
+
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#hasCuepoint>
+    rdf:type	    rdf:Property ;
+    rdfs:range	    <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint> ;
+    rdfs:domain     ebucore:Asset ;
+    rdfs:label	    "hasCuepoint"@en ;
+    rdfs:comment    "Property to associate an asset with its cuepoint metadata."@en .
+
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#hasCuepointType>
+    rdf:type      rdf:Property ;
+    rdfs:domain   <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint> ;
+    rdfs:range    <http://www.w3.org/2001/XMLSchema#string> ;
+    rdfs:comment  "A type of Cuepoint."@en ;
+    rdfs:range    <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint_Type> ;
+    rdfs:range    <http://www.w3.org/2001/XMLSchema#anyURI> ;
+    rdfs:label    "Cuepoint type"@en .
+
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint_Type>
+    rdf:type         rdfs:Class ;
+    rdfs:comment     "To define a type or cuepoint."@en ;
+    rdfs:subClassOf  skos:Concept ;
+    rdfs:label       "Cuepoint type"@en .
+
+<https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#cuepointTimestamp>
+    rdf:type      rdf:Property ;
+    rdfs:domain   <https://metadata.amagi.tv/ebu/amagi_ebucoreExtension#Cuepoint> ;
+    rdfs:range    <http://www.w3.org/2001/XMLSchema#string> ;
+    rdfs:comment  "The time when the Cuepoint occurs."@en ;
```

Comment:
Give more details. example of the string value and the fact that float conversion should work

Reply:
Okay @shashi-banger will add more details